webots_ros2_epuck/EPUCK_ROS2.md

```diff
@@ -0,0 +1,206 @@
+# ROS2 for E-Puck
+Here you will find instructions on how to use utilize e-puck ROS2 API. 
+
+This documentation is common for for physical ([`epuck_ros2`](https://github.com/cyberbotics/epuck_ros2)) and simulated ([`webots_ros2_epuck`](https://github.com/cyberbotics/webots_ros2/tree/master/webots_ros2_epuck)) robot, so there is a slight difference in using the launch file.
+Launch file `example_launch.py` runs the Webots simulation and includes `example_tools_launch.py`.
+Therefore, you can use the launch file `example_tools_launch.py` for the physical robot with the same parameters as you would `example_launch.py` for the simulated robot.
+To emphasis this, you will see `[tools_]` in the commands which indicates you can use `example_tools_launch.py` and `example_launch.py` depending on whether you work with the physical or simulated robot, e.g.:
+```
+ros2 launch webots_ros2_epuck example_[tools_]launch.py rviz:=true
+```
+
+## Infra-red, Light Sensors, and LEDs
+![e-puck2 camera and infrared sensors](https://raw.githubusercontent.com/cyberbotics/webots/master/docs/guide/images/robots/epuck/sensors_and_leds.png)  
```

Comment:
I would rather re-upload this image in this repo as it might easily be broken by changes in the other repo.